CoherenceSpectrogramViewer/Source/CoherenceVisualizer.cpp

```diff
@@ -988,7 +992,7 @@ void CoherenceVisualizer::endAnimation()
 	{
 		group2Buttons[i]->setEnabled(true);
 	}
-	if (IsSpectrogram == false)
+	if (IsSpectrogram == false) // A lot of these are the same between the if/else. I'd take those out to make it more clear what the difference is here.

```

Comment:
Again, just coding style. Looks of overlap in this if/then that should be looked at.